CredentialCardDetail.vue

```diff
@@ -31,6 +31,7 @@
             {{issuerName}}
           </q-item-section>
         </q-item-label>
+        <slot name="profile" />
```

Comment:
The credential card has no notions of profile. This should be renamed to something more general. What part of the card is this? We could name this `footer` or `bottom` or something else.